.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: python
 cache: pip
 python:
-  - "nightly" # currently points to 3.6-dev
+  # see https://docs.travis-ci.com/user/languages/python/ for options
+  - "nightly"
```

Comment:
Right 3.7 is the same as 3.6-dev. Was hoping to tie it to 3.6 beta instead, but since that's not a travis option yet I figured it would be okay to wait til 3.6 final